### PR TITLE
NIFIREG-67 Revisions to Admin Guide

### PR DESCRIPTION
**Pull request by kevdoran**
Minor edits and corrections to the Admin Guide, particularly
related to configuring an instance with security options.

### CODE REVIEW

File: nifi-registry-docs/src/main/asciidoc/administration-guide.adoc [hunk 1]
```diff
@@ -237,9 +237,12 @@ During startup there is a check to ensure that there are no two users/groups wit
...
-* Initial User Identity - The identity of a users and systems to seed the Users File. The name of each property must be unique, for example: "Initial User Identity A", "Initial User Identity B", "Initial User Identity C" or "Initial User Identity 1", "Initial User Identity 2", "Initial User Identity 3"
+* Users File - The file where the FileUserGroupProvider stores users and groups.
+  By default, 'users.xml' in the 'conf' directory is chosen.
+* Initial User Identity - The identity of a user or system to seed an empty Users File.
+  Multiple Initial User Identity properties can be specified, but the name of each property must be unique, for example: "Initial User Identity A", "Initial User Identity B", "Initial User Identity C" or "Initial User Identity 1", "Initial User Identity 2", "Initial User Identity 3"
+
+NOTE: Initial User Identities these identities are only created if the specified Users File is missing or empty during NiFi Registry startup. Changes to the configured Initial Users Identities will not take effect if the Users File is populated.
```
Comment (andrewmlim) [inline nifi-registry-docs/src/main/asciidoc/administration-guide.adoc]: Remove "these identities"

File: nifi-registry-docs/src/main/asciidoc/administration-guide.adoc [hunk 2]
```diff
@@ -287,34 +290,30 @@ The CompositeConfigurableUserGroupProvider will provide support for retrieving u
 * Configurable User Group Provider - A configurable user group provider.
 * User Group Provider - The identifier of user group providers to load from. The name of each property must be unique, for example: "User Group Provider A", "User Group Provider B", "User Group Provider C" or "User Group Provider 1", "User Group Provider 2", "User Group Provider 3"
 
+After you have configured a UserGroupProvider, you configure an AccessPolicyProvider that will control Access Policies for the identities in the UserGroupProvider.
```
Comment (andrewmlim) [inline nifi-registry-docs/src/main/asciidoc/administration-guide.adoc]: To be consistent with rest of doc, add "must" in "After you have configured...., you must configure an AccessPolicyProvider..."

File: nifi-registry-docs/src/main/asciidoc/administration-guide.adoc (hunk 1 above)
Comment (kevdoran) [reply nifi-registry-docs/src/main/asciidoc/administration-guide.adoc]: thanks, will do

**Comment by kevdoran** [comment]
Updated to address the corrections noted by @andrewmlim. Thanks!

**Comment by bbende** [comment]
Looks good, merging...

**Pull request closed without merging**